path.c

```diff
@@ -1232,6 +1232,11 @@ int change_directory(
 
     assert(!logical || !ensure_pwd);
 
+    if ( newpwd[0] == L'\0' ) {
```

Comment:
Although we don't have an explicit coding style guide, we usually don't put spaces inside parentheses.

```suggestion
    if (newpwd[0] == L'\0') {
```